drivers/sx127x/sx127x_internal.c

```diff
@@ -122,7 +122,7 @@ void sx1276_rx_chain_calibration(sx127x_t *dev)
     reg_pa_config_init_val = sx127x_reg_read(dev, SX127X_REG_PACONFIG);
     initial_freq = (double) (((uint32_t) sx127x_reg_read(dev, SX127X_REG_FRFMSB) << 16)
                              | ((uint32_t) sx127x_reg_read(dev, SX127X_REG_FRFMID) << 8)
-                             | ((uint32_t) sx127x_reg_read(dev, SX127X_REG_FRFLSB))) * (double)LORA_FREQUENCY_RESOLUTION_DEFAULT;
+                             | ((uint32_t) sx127x_reg_read(dev, SX127X_REG_FRFLSB))) * (double)SX127X_FREQUENCY_STEP;
```

Comment:
```suggestion
                             | ((uint32_t) sx127x_reg_read(dev, SX127X_REG_FRFLSB)));
```

Reply:
I just notice this operation is there to save and restore the chanel freq. It makes no sense to convert it to channel representation and then again to hw values. With https://github.com/RIOT-OS/RIOT/pull/13318/commits/a3b14375137f9513f8d1e21864f5d013b2a1a74d I just save and restore register values.